### ISSUE TITLE
Unable to access GitHub Copilot Chat

### ISSUE THREAD

**Issue by ahsancloudcode** [microsoft/vscode-copilot-release#333]

**Comment by jeffaxup** [comment]
+1 on this.
I am paying for a CoPilot subscription currently, and I signed up for it expressly to try to out the Chat feature.
Hidden at the end of the plug-in description is a small field "Accepted to GitHub Copilot chat private beta (join the [waitlist](https://github.com/github-copilot/chat_waitlist_signup/join))."

- This was not mentioned up front at the time of purchasing CoPilot.
- I have been waiting for weeks for access.
- CoPilot is really not all that useful. It frequently makes incorrect suggestions, distracts you while you code, and doesn't help you achieve larger goals or write code for multiple parts of a file at once.
- In short, Copilot Chat is the only thing that is going to be useful for me.
- Copilot Chat needs to drop their "beta signup" requirement and just open this up as an opt-in feature to all CoPilot subscribers.

It also needs to be much clear (at the top of the main CoPilot subscription) that it currently doesn't include Chat.
I will likely be closing my CoPilot account as there is no reason to pay a monthly fee for the functionality it currently has.

Please inform the CoPilot PM that this is not being handled well.

**Comment by ahsancloudcode** [comment]
Hi @jeffaxup I am already in waitlist of copilot chat , this time it is showing me a message which is 

" Thanks for joining!
You’re already on the waitlist for GitHub Copilot chat private beta!
We’ll email you when we’ve enabled it on your account."

So, kindly tell me how much time is required for me to access Copilot chat. I joined the waitlist almost about 4 months ago.

**Comment by anthonykim1** [comment]
Should be resolved now, let us know if not
